scene-format.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+  message Object {
+    int32 id;
+    string reference;         // path to file or url
+  }
+  message Material {
+    int32 object_id;
+    string type;
```

Comment:
The same question as for lights: how will be new parameters be added? It is crucial for materials because they will use different parameters. For example, albedo, texture, normal texture, smoothness, specular, metallic, and many other parameters can be used in different variations.